.sqlfluff

```diff
@@ -0,0 +1,2 @@
+[sqlfluff]
+dialect = ansi
```

Comment:
I’m not sure if this is going to make it into the final PR, or if this is temporary for now but I’m not loving this for a number of reasons:

1. it feels like a bit of a hack. You must specify a dialect, but we don’t really. It teaches people developing not to specify a dialect.
2. When running SQL from this the repo it will work, but running elsewhere will fail. It will be confusing.
3. I often add a specific `.sqlfluff` config file while developing and testing but definitively do not want that checked in but that might happen now. Though guessing testing would pick it up.

However I do recognise how many places you’re likely to need to specify a dialect without this :-( Luckily for dialect test fixtures you don’t need to as we have one for each of those, but for lots of other test files you might now need to.

Reply:
Just trying to get the tests to pass now. Will look for better ways to do things once they do.

Reply:
I'll keep SQLFLUFF_TESTENV in mind. Ideally, we can get all the tests working with changes on the test side, minimizing changes to the production code (i.e. stuff in `src/sqlfluff/`).